src/design-tokens/mixins.css

```diff
@@ -120,7 +120,7 @@
    */
   &:disabled,
   &:disabled::placeholder {
-    border-color: var(--eds-theme-color-form-border-disabled);
+    border-color: var(--eds-theme-color-border-disabled);
```

Comment:
In https://github.com/chanzuckerberg/edu-design-system/pull/1279 I darkened the `color-border-disabled` token for buttons and added a separate `color-form-border-disabled` token here to avoid changing the input field styles until I talked to design about it. Since the input field disabled border is now in sync with the others, I removed this token and went back to using the general `color-border-disabled` token in the input field styles.